schema/V1__reference_schema.sql

```diff
@@ -95,6 +95,18 @@ CREATE TABLE dog_vet_preferences (
   CONSTRAINT dog_vet_preferences_pk PRIMARY KEY (dog_id, vet_id)
 );
 
+CREATE TYPE t_call_outcome AS ENUM ('APPOINTMENT', 'DECLINED', 'OPT_OUT');
+
+CREATE TABLE calls (
+  call_id BIGSERIAL,
+  call_creation_time TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  vet_id BIGINT NOT NULL,
+  dog_id BIGINT NOT NULL,
```

Comment:
Foreign key constraints are not specified.

Reply:
Fixed in https://github.com/barkbank/barkbank-schema/pull/20/commits/c0cc6e8ce6a09f54fddb3e4c18759d3408dfaea3